sycl/test-e2e/ESIMD/local_accessor_block_load_store.cpp

```diff
@@ -12,8 +12,6 @@
 // UNSUPPORTED: gpu
 //
 // TODO: Enable the test when GPU driver is ready/fixed.
-// XFAIL: opencl || windows || gpu-intel-pvc
```

Comment:
I wanted to confirm it intentional change here. Does the test for local_accessor passes on all platforms now?
Or "UNSUPPORTED: gpu" is missing here?

Reply:
Thank you. I did not see that it is on the Line 12.
If you remove L15, then L14 needs to be removed as well. +Same for few other places.